src/databricks/labs/remorph/jvmproxy.py

```diff
@@ -18,6 +18,7 @@ def fn(**_):
 
 
 class JvmProxy:
+    # TODO Refactor this class to use morpheus instead
```

Comment:
This is no longer required. Morpheus is a LSP server running in its own process, communication is done via LSP protocol

Reply:
Agreed, I suspect a transition time where we will need this to support or trigger CLI should be removed when LSP communication is available for Morpheus.